### ISSUE TITLE
bug: [Bun] variable not found in script

### ISSUE THREAD

**Issue by vrachev** [windmill-labs/windmill#2040]
### Describe the bug

Happening in the windmill cloud app. 

**Error on Bun only, works normally on Deno.**

Trying to embed a variable inside a script eg:
```
  gsheets_auth = await getVariable('u/vrachev/gsheets');
```

Getting error:
```
Error: Variable not found at u/vrachev/gsheets or not visible to you
```

If I instead run a test execution and create a parameter for the variable, it succeeds when I input `u/vrachev/gsheets`. 

### To reproduce

1.  Embed existing workspace variable in script eg:
```
  gsheets_auth = await getVariable('u/vrachev/gsheets');
```
2. Run test execution.

### Expected behavior

Variable to be imported correctly

### Application version

EE v1.145.3-1-g4dcf7ae0

**Comment by rubenfiszel** [comment]
There is a few issues with bun including this one that I am solving with a big update to bun today. Thank you for the report.

**Comment by vrachev** [comment]
Appreciate it.

**Comment by rubenfiszel** [comment]
Fixed starting in 1.146.1
